From 85c76fdd21b1a472263db6613a50fc9cd4abe983 Mon Sep 17 00:00:00 2001
From: "Marco A. Gutierrez" <marcogg@marcogg.com>
Date: Fri, 2 Oct 2020 04:08:49 +0000
Subject: [PATCH] removing check for ros env variable

---
 CMakeLists.txt | 9 ++-------
 1 file changed, 2 insertions(+), 7 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index c8dfa36..72a9667 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -87,13 +87,8 @@ target_compile_definitions(${PROJECT_NAME} PRIVATE "RVIZ_DEFAULT_PLUGINS_BUILDIN
 pluginlib_export_plugin_description_file(rviz_common plugin_description.xml)
 
 ament_export_include_directories(include)
-if($ENV{ROS_DISTRO} STREQUAL foxy)
-  ament_export_targets(${PROJECT_NAME} HAS_LIBRARY_TARGET)
-elseif($ENV{ROS_DISTRO} STREQUAL eloquent)
-  ament_export_interfaces(${PROJECT_NAME} HAS_LIBRARY_TARGET)
-else()
-  message(FATAL_ERROR "Unsupported ROS distribution")
-endif()
+ament_export_targets(${PROJECT_NAME} HAS_LIBRARY_TARGET)
+
 ament_export_dependencies(
   Qt5
   rviz_common
-- 
2.25.1

